client/directives/file-filter.js

```diff
@@ -0,0 +1,40 @@
+const isFileInput = el => el.tagName === 'INPUT' || el.type === 'file';
+const isProduction = process.env.NODE_ENV === 'production';
+
+export const install = Vue => {
+  const warn = (msg, vm) => Vue.util.warn(`[v-filefilter]: ${msg}`, vm);
+  Vue.directive('filefilter', {
```

Comment:
Directives should be named using `kebab-case`.

Reply:
True that but I'm soooo _satisfied_ with looks of `v-filefilter` 😏